syntax = "proto3";

option java_package = "com.terminus.ble.common.protobuf";
package TslBLEProto;

// 获取设备信息
message TSLBLEDeviceInfoRequest {
    int32 security = 1;     // 密码， BCD码表示
}

// 返回设备信息
message TSLBLEDeviceInfoResponse {
    int32   security = 1;               // 密码， BCD码表示
    string  sn = 2;                     // 设备序列号
    int32   dev_type = 3;               // 设备类型
    string   manufacturers = 4;              
    string  dev_name = 5;               // 设备名字
    string  software_version = 6;       // 软件版本号
    string  hardware_version = 7;       // 硬件版本号
	bytes   device_code = 8; 
	bytes   location_code = 9; 
}
